mungegithub/mungers/blunderbuss.go

```diff
@@ -90,7 +93,11 @@ func printChance(owners weightMap, total int64) {
 	}
 }
 
-func getPotentialOwners(author string, feats *features.Features, files []*githubapi.CommitFile, leafOnly bool) (weightMap, int64) {
+type ownerVetter interface {
```

Comment:
The size of `config` (a pointer) is actually less than that of an interface since an interface's internal representation is a struct with 2 pointers. In either case the size of the data getting passed is not a concern.
If you are just using the interface to limit the access that `getPotentialOwners` gets to the config, you should switch to passing just the function instead of an interface containing the function since you don't need an object with an IsAssignee function, just the function itself.